experimental/adaptive-tool/README.md

```diff
@@ -1,74 +1,125 @@
-1. Open an LG (.lg) or LU (.lu) file and the adaptive-tool extension will activate.
-
-## Install From Local VSIX File
-1. [Build VSIX from code.](#buildPackage)
-1. Open Visual Studio Vode and navigate to the **Extensions**.
-1. Click the three dots in the top right corner of the Extensions. Then click **Install from VSIX...** and select the VSIX file you created in the first step.
-
-# Features
-
-## Language Features
-Language features are driven by the [language server protocol](./languageServer.md)
-
-### Syntax highlighting, diagnostic check, auto-suggest, functionality
+-->
+
+| Feature                                       | File type supported |
+|-----------------------------------------------|---------------------|
+| [Syntax highlighting](#syntax-highlighting-diagnostic-checks-and-autocompletion)                           | LG, LU              |
+| [Diagnostic checks](#syntax-highlighting-diagnostic-checks-and-autocompletion)                             | LG, LU              |
+| [Autocompletion](#syntax-highlighting-diagnostic-checks-and-autocompletion)                                | LG, LU              |
+| [Template and function hover](#hover-suggestions-and-navigation)                   | LG                  |
+| [Template definition](#hover-suggestions-and-navigation)                           | LG                  |
+| [Template, function, and structure suggestions](#hover-suggestions-and-navigation) | LG                  |
+| [Debugging](#debugging)                                     | LG, LU, Dialog      |
+
+## Getting started
+
+Adaptive Tool can be installed [from the Visual Studio Marketplace](#install-from-the-visual-studio-marketplace) or [from a local VSIX file](#install-from-a-local-vsix-file).
+
+### Install from the Visual Studio Marketplace
+
+1. Install the [Bot Framework Adaptive Tool](https://marketplace.visualstudio.com/items?itemName=adaptive-tool) from the [VS Code](https://marketplace.visualstudio.com/vscode) tab of the Visual Studio Marketplace.
+1. Open an LG, LU, or dialog file and the extension will activate.
+
+### Install from a local VSIX file
+
+1. Clone the [BotBuilder-Samples](https://github.com/microsoft/BotBuilder-Samples) repo.
+1. In a terminal or command prompt, navigate to the directory containing the Adaptive Tool, **experimental/adaptive-tool**, and run the following commands:
+
+    ```cmd
+    npm install
+    ```
+    
+    ```cmd
+    npm run build
+    ```
+    
+  1. Run the command below to install [vsce](https://www.npmjs.com/package/vsce):
+  
+      ```cmd
+      npm install -g vsce
+      ```
+    
+1. Finally, run the following command to export the VSIX file:
+
+    ```cmd
+    vsce package
+    ```
+    
+    You're now ready to install Adaptive Tool.
+1. Open VS Code and open the **Extensions** view.
+1. Click the **More Actions** button (...) in the upper-right corner of the window. Then click **Install from VSIX..** and select the VSIX file you created earlier.
```

Comment:
I didn't notice the first time through that there are at least two different ellipsis buttons visible. Perhaps:
```suggestion
1. Click the **Views and More Actions** button (...) in the upper-right corner of the Extensions panel. Then click **Install from VSIX..** and select the VSIX file you created earlier.
```